### ISSUE TITLE
No updated for patch #2?

### ISSUE THREAD

**Issue by disgusdad** [synpoox/pd2-reawakening#29]
Great mod, noticed while applying it to latest PD2 (patch 2) the ALT (show items) is broken again with this mod installed.

**Comment by synpoox** [comment]
Yeah, haven't been able to figure out why so far, still looking into it
